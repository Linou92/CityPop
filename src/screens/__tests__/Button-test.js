import React from "react";
import renderer from "react-test-renderer"
import Button from "../../components/Buttons"

test('renders button correctly', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
});