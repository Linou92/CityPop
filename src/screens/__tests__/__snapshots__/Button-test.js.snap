// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Buttons correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#50aab5",
      "borderRadius": 10,
      "flexDirection": "row",
      "height": 60,
      "opacity": 1,
      "width": "60%",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 15,
          "textAlign": "center",
        }
      }
    >
       
       
    </Text>
  </View>
</View>
`;
